### ISSUE TITLE
How to `--dump` to file?

### ISSUE THREAD

**Issue by tooomm** [lycheeverse/lychee-action#212]
It looks like `--dump` only prints the found links to the console.

I would like to dump the output to a file to further work with it.
Not sure how to do this with the action.

`output:` is not considered

**Comment by mre** [comment]
Yeah, that's a bug in lychee then. It should consider the output setting. Let me move that over to the main repo.
